CHANGELOG.md

```diff
@@ -20,6 +20,14 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
   not suffer from the aforementioned limitations. We believe that this change
   should not break existing taint rules, but please report any regressions that
   you may find.
+- taint-mode: Let's say that e.g. `sanitize(x)` sanitizes `x` by side-effect.
+  Previously, we had to rely on a trick that declared that _any_ occurrence of
+  `x` inside `sanitize(x); ...` was sanitized. If `x` later overwritten with
+  tainted data, the taint engine would still regard `x` as safe. Now, if you
+  specify that the code variable itself is sanitized, the taint engine will
```

Comment:
you could add "(thanks to the new 'focus-metavariable:' feature introduced in 0.9??)"